book/http.md

```diff
@@ -78,8 +78,8 @@ telnet example.org 80
 ```
 
 ::: {.web-only}
-(Note: When you see a gray outline, it means that the code in question is
-an example only, and *not* actually part of our browser's code.)
+(Note: when a code block has an "Example Code" tag, that means that
```

Comment:
The "Trying 93.184.216.34..." block in this file is also output. Maybe there are more beyond that?